Wave/Wave/RowType.swift

```diff
@@ -13,4 +13,5 @@ public protocol RowType {
     
     func registerCells(tableView: UITableView)
     func configureCell(tableViewCell: UITableViewCell)
+    func didSelectCell(tableViewCell: UITableViewCell, tableView: UITableView, indexPath: NSIndexPath)
```

Comment:
Oh wait on second though we do need `indexPath` because the super class calls `tableView.deselectRowAtIndexPath()`